packages/app/src/debug/DebugFailedTest.vue

```diff
@@ -30,7 +30,17 @@
           titlePart.type === 'LAST-1' ? 'shrink-0 whitespace-pre' :
           titlePart.type === 'LAST-0' ? 'pl-2.5 truncate' : 'px-2.5 truncate'"
       >
-        {{ titlePart.title }}
+        <template v-if="titlePart.title === '...'">
```

Comment:
```suggestion
        <template v-if="titlePart.type === 'ELLIPSIS'">
```
Check with type instead of title